File: java/PlanesAndroid/app/src/test/java/HeadDataTest.java
import com.google.common.truth.Truth;
import com.planes.common.GuessPoint;
import com.planes.common.HeadData;
import com.planes.common.Type;

import org.junit.Test;
import static com.google.common.truth.Truth.assertThat;

public class HeadDataTest {

    @Test
    public void HeadData_Init() {
        HeadData hd = new HeadData(10, 10, 5, 5);
        hd.update(new GuessPoint(5,5, Type.Dead));
    }
}

File: kotlin/PlanesAndroid/app/src/test/java/HeadDataTest.kt
package com.planes.test
import com.planes.single_player_engine.GuessPoint
import com.planes.single_player_engine.HeadData
import com.planes.single_player_engine.Type
import org.junit.Test

class HeadDataTest {
    @Test
    fun HeadData_Init() {
        val hd = HeadData(10, 10, 5, 5)
        hd.update(GuessPoint(5, 5, Type.Dead))
    }
}